PF/Assignment1/anshitaarya/prg2.java

```diff
@@ -0,0 +1,41 @@
+package anshitaarya;
+
+public class prg2 {
+	 public static int[] removeDuplicates(int[] duplicate){
+         int l= duplicate.length;
+         int i,j,m=1,count;
+        
+         int[] dup= new int[l];
```

Comment:
ACTION: This array will contain all unique elements then size of this array must not be 1.